ts/features/messages/components/MessageAttachmentPreview.tsx

```diff
@@ -128,20 +130,23 @@ type Props = {
 
 export const MessageAttachmentPreview = (props: Props): React.ReactElement => {
   const [isError, setIsError] = useState(false);
-
   const messageId = props.messageId;
   const attachmentId = props.attachmentId;
   const downloadPot = useIOSelector(state =>
-    mvlDownloadFromAttachmentSelector(state, { messageId, id: attachmentId })
+    downloadFromAttachmentSelector(state, { messageId, id: attachmentId })
   );
   const download = pot.toUndefined(downloadPot);
+
   return download ? (
     <BaseScreenComponent
       goBack={true}
       contextualHelp={emptyContextualHelp}
       headerTitle={I18n.t("features.mvl.details.attachments.pdfPreview.title")}
     >
-      <SafeAreaView style={styles.container} testID={"MvlDetailsScreen"}>
+      <SafeAreaView
+        style={styles.container}
+        testID={"message-attachment-preview"}
```

Comment:
Has this testId changed on 'ts/features/mvl/screens/__test__/MvlRouterScreen.test.tsx', line 65?

Reply:
I didn't change it because the test expects the id `MvlDetailsScreen` in `ts/features/mvl/screens/details/MvlDetailsScreen.tsx`